docs/pipeline-components-and-applications/enrichment-components/stream-enrich/index.md

```diff
@@ -1,12 +1,17 @@
 ---
-title: "Stream Enrich"
+title: "stream-enrich"
 date: "2020-11-09"
 sidebar_position: 800
```

Comment:
Seems a bit confusing when it’s below the other Enrich apps (even though I know it’s because it’s about to be deprecated). Can we move it next to them?

Reply:
You mean before above `Monitoring` and `Configuration` ? These pages are not for `Stream Enrich` assets, only new ones.